doc/man/8/ceph-deploy.rst

```diff
@@ -251,93 +247,48 @@ Subcommand ``list`` lists disk partitions and Ceph OSDs.
-start with ``ceph disk activate-all``.
-
-Usage::
-
-	ceph-deploy disk activate [HOST:[DISK]]
-
-Here, [HOST] is hostname of the node and [DISK] is disk name or path.
-
-Subcommand ``zap`` zaps/erases/destroys a device's partition table and contents.
-It actually uses ``sgdisk`` and it's option ``--zap-all`` to destroy both GPT and
-MBR data structures so that the disk becomes suitable for repartitioning.
-``sgdisk`` then uses ``--mbrtogpt`` to convert the MBR or BSD disklabel disk to a
-GPT disk. The ``prepare`` subcommand can now be executed which will create a new
-GPT partition.
-
-Usage::
-
-	ceph-deploy disk zap [HOST:[DISK]]
+	ceph-deploy disk list HOST
 
-Here, [HOST] is hostname of the node and [DISK] is disk name or path.
 
+Subcommand ``zap`` zaps/erases/destroys a device's partition table and
+contents.  It actually uses ``ceph-volume lvm zap`` remotely, alternatively
+allowing someone to remove the Ceph metadata from the logical volume.
 
 osd
 ---
 
 Manage OSDs by preparing data disk on remote host. ``osd`` makes use of certain
 subcommands for managing OSDs.
 
-Subcommand ``prepare`` prepares a directory, disk or drive for a Ceph OSD. It
-first checks against multiple OSDs getting created and warns about the
-possibility of more than the recommended which would cause issues with max
-allowed PIDs in a system. It then reads the bootstrap-osd key for the cluster or
-writes the bootstrap key if not found. It then uses :program:`ceph-disk`
-utility's ``prepare`` subcommand to prepare the disk, journal and deploy the OSD
-on the desired host. Once prepared, it gives some time to the OSD to settle and
-checks for any possible errors and if found, reports to the user.
+Subcommand ``create`` prepares a device a Ceph OSD. It first checks against
+multiple OSDs getting created and warns about the possibility of more than the
+recommended which would cause issues with max allowed PIDs in a system. It then
+reads the bootstrap-osd key for the cluster or writes the bootstrap key if not
+found.
+It then uses :program:`ceph-volume` utility's ``lvm create`` subcommand to
+prepare the disk, (and journal if using filestore) and deploy the OSD on the desired host.
+Once prepared, it gives some time to the OSD to start and checks for any
+possible errors and if found, reports to the user.
 
-Usage::
-
-	ceph-deploy osd prepare HOST:DISK[:JOURNAL] [HOST:DISK[:JOURNAL]...]
+Bluestore Usage::
 
-Subcommand ``activate`` activates the OSD prepared using ``prepare`` subcommand.
-It actually uses :program:`ceph-disk` utility's ``activate`` subcommand with
-appropriate init type based on distro to activate the OSD. Once activated, it
-gives some time to the OSD to start and checks for any possible errors and if
-found, reports to the user. It checks the status of the prepared OSD, checks the
-OSD tree and makes sure the OSDs are up and in.
+	ceph-deploy osd create --data DISK HOST
```

Comment:
We could add a line that explains the other 2 optional parameters

"--block-db /path/to/db-device --block-wal /path/to/wal-device"

Reply:
I am not adding information on every flag, but I did add that "other flags available, please see the man page or the --help menu on ceph-deploy osd create" which I think should suffice